rsocket-ipc-jackson/src/main/java/io/rsocket/ipc/marshallers/Json.java

```diff
@@ -0,0 +1,56 @@
+package io.rsocket.ipc.marshallers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.module.afterburner.AfterburnerModule;
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.ByteBufAllocator;
+import io.netty.buffer.ByteBufInputStream;
+import io.netty.buffer.ByteBufOutputStream;
+import io.rsocket.ipc.Marshaller;
+import io.rsocket.ipc.Unmarshaller;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import reactor.core.Exceptions;
+
+public class Json {
+  private final ObjectMapper mapper;
+
+  private Json() {
+    mapper = new ObjectMapper();
+    mapper.registerModule(new AfterburnerModule());
```

Comment:
IIUC this shouldn't change behaviour, just implements common optimisations for Jackson on POJOs.